source/includes/getting-started/_lets-get-started.html.md

```diff
@@ -1,32 +1,72 @@
 # Let's Get Started!
 
-Before you begin, you will need to have access to our Beta. If you don't have access yet, request access at [bons.ai][1].
+Before you begin, you will need to have access to our Beta. If you don't have access yet, request access at [bons.ai][1] and then come visit us in the Forums and let us know why you're excited to try out Bonsai!
 
-## Install Prerequisites
+The next section walks you through how to install Python, Git, and Pip, but if you're already ahead of the curve and have them all installed, skip down to [Setup the Bonsai CLI][2]. 
 
-If you have Anaconda installed on your system, go to Step 1 of our [Getting Started with Anaconda][2] guide instead.
+**Note:** We highly recommend Mac users use HomeBrew or download Python directly in the General Install instructions so they can perform the following steps on user-land Python rather than [system Python][13].
 
-### Windows
+# Install Prerequisites
 
-1. Download and install Python from [python.org][3].
-2. Install Git (e.g. via GitHub desktop for windows).
-3. You may need to reboot your computer after installing Python and Git to make sure all install settings take effect.
+If you have Anaconda installed on your system, skip down to [Install with Anaconda][9].
 
-> Mac OS
+If you *do not* have Anaconda installed on your system continue to follow our **General Install** instructions.
+
+## General Install
+
+### Windows
+
+> [Chocolatey][14] commands
 
 ```shell
-sudo easy_install pip
+choco install git
+choco install python # or you can install python2 if you need version 2.7
 ```
 
+1. **Git**: Download and install Git from [Git for Windows][11] or with [Chocolatey][14].
+2. **Python**: Download and install Python from [python.org][3] or with [Chocolatey][14].
+3. **Pip**: Python version 2.7.9 and greater come with pip, but if for some reason you need to use a different version of python please follow [these instructions][12].
+
+**Note:** You may need to reboot your computer after installing Python and Git to make sure all install settings take effect.
+
 ### Mac OS X
 
-1. Install pip
+> [Homebrew][6] commands
+
+```shell
+brew install git
+brew install python3 # or you can install python if you need version 2.7
+```
+
+1. **Git**: Git is more than likely already installed on your computer if you have Xcode, but if not, run the git command or install with [Homebrew][6].
+2. **Python**: Download and install Python from [python.org][3] or with [Homebrew][6].
+3. **Pip**: Python version 2.7.9 and greater come with pip, but if for some reason you need to use a different version of python please follow [these instructions][12].
+
+## Install with Anaconda
+
+If you *do not* have Anaconda installed on your system, please skip this section.
```

Comment:
Could you elaborate on what the process is for Anaconda? Is it the same as regular Windows and Mac behavior and the Anaconda specific instructions take place after that?